.metals/metals.lock.db

```diff
@@ -0,0 +1,6 @@
+#FileLock
```

Comment:
What is this file?

Reply:
and the metals.h2.db?

Reply:
scala files generated automatically by an extension, doesn't matter delete or leave